layouts/partials/hooks/head-end.html

```diff
@@ -0,0 +1,4 @@
+{{/* Include link to blog canonical URL, if "canonical_url" field exists in front matter. */}}
```

Comment:
I'd drop this comment line, because to me, the code speaks for itself (and besides, this will work for any page, not just the blog).